fe/fe-core/src/main/jflex/sql_scanner.flex

```diff
@@ -210,6 +210,8 @@ import com.starrocks.qe.SqlModeHelper;
         keywordMap.put("grant", new Integer(SqlParserSymbols.KW_GRANT));
         keywordMap.put("grants", new Integer(SqlParserSymbols.KW_GRANTS));
         keywordMap.put("group", new Integer(SqlParserSymbols.KW_GROUP));
+        keywordMap.put("resource_group", new Integer(SqlParserSymbols.KW_RESOURCE_GROUP));
```

Comment:
I think we should unify the name. either `resource_group` or `work_group `

Reply:
resource_group is just used in sql stmt, which  product manager prefers to